src/components/LocationSelector/utils.ts

```diff
@@ -32,3 +32,22 @@ export const findCenter = (geocoder, placeId) =>
             }
         });
     });
+
+export const getAddressComponents = (geocoder, location) =>
+    new Promise(function geocoderPromise(resolve, reject) {
+        geocoder.geocode({ location }, (results, status) => {
```

Comment:
Just curious, what is idea write here named function?:
```
function geocoderPromise(resolve, reject) {
```
It can be:
```
(resolve, reject) => {
```